services/infrastructure/mongo.initdb.js

```diff
@@ -0,0 +1,32 @@
+db.api_clients.insert({
```

Comment:
oh, i'm not sure i like the fact if we make changes to the test data we have to update the Frontend AND the API. Hopefully this will only be for the short term, but something we need to keep an eye on.

What I would say is that I expect a lot of the frontend integrations to also be required by the API when it comes to validation purposes i.e. the API will need to fetch the company profile to ensure that the Company is still "active", it will need to fetch the directors etc etc.

Knowing this, is it still worth maintaining two separate docker-compose files for Web and API? We know our docker-compose solution isn't a long term one anyway, so for now why don't we just keep one compose file for everything?

What are your thought s @naufalmirk @mateuszc-kainos ?